assets/src/application/ui/display/ColorSwatch/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import './style.scss';
+
+interface Props {
+	className?: string;
+	label?: string;
+}
+
+const ColorSwatch: React.FC<Props> = ({ className, label }) => {
```

Comment:
I thought that `application/ui/display` is a good place for this component.

Reply:
ya that's perfect